packages/i18nify-js/src/modules/geo/data/listOfAllCountries.ts

```diff
@@ -220,11 +251,15 @@ export const LIST_OF_ALL_COUNTRIES: CountryCodeType[] = [
   'VI',
   'VN',
   'VU',
+  'WF',
   'WS',
-  'XK',
   'YE',
+  'YT-UNF',
   'YT',
   'ZA',
   'ZM',
   'ZW',
 ];
+//226
+
+// less 32
```

Comment:
what is this ?

Reply:
Was doing some calculation, forgot to remove this.